WooCommerce/WooCommerceScreenshots/WooCommerceScreenshots.swift

```diff
@@ -26,9 +26,8 @@ class WooCommerceScreenshots: XCTestCase {
         app.launchArguments.append("mocked-network-layer")
         app.launchArguments.append("-simulate-stripe-card-reader")
         app.launchArguments.append("disable-animations")
-        app.launchArguments.append("-mocks-port")
         app.launchArguments.append("-mocks-push-notification")
-        app.launchArguments.append("\(server.listenAddress.port)")
+        app.launchArguments.append(contentsOf: ["-mocks-port", "\(server.listenAddress.port)"])
```

Comment:
The issue here was that when we added the launch argument `-mocks-push-notification`, it was accidentally being  put between the key `mocks-port` and its value. So to avoid a similar thing in the future, I'm now appending the key and its value both in a single line.